transcom/mymove
# MB2441 Fix accent container for requested shipment section move details

## Description

Missed different HHG shipment types. This includes additional HHG shipment types `HHG_SHORTHAUL_DOMESTIC` and `HHG_LONGHAUL_DOMESTIC` to the shipment container and shipment display component. This will properly show the accented styling for the container.

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

To test:
1. Log in to office app (locally)
2. Navigate to `http://officelocal:3000/too/customer-moves/`
3. Click on any moves
4. For this particular move order, change the `shipment_type` column value in the `mto_shipments` table DB to another HHG type. For example, `HHG_LONGHAUL_DOMESTIC`.
5. Refresh page to see changes

## Code Review Verification Steps


## References

* [Jira story](https://dp3.atlassian.net/browse/MB-2441) for this change

## Screenshots

![image](https://user-images.githubusercontent.com/1522549/83299732-9769e300-a1ab-11ea-83ea-2cd967ca7f5e.png)
